From 00aa141fae192860906ba40594b5f18ae4b68470 Mon Sep 17 00:00:00 2001
From: Vasily Khoruzhick <anarsoul@gmail.com>
Date: Sat, 23 Jun 2018 22:11:39 -0700
Subject: [PATCH 03/10] video: anx6345: don't fail if there's no sleep or reset
 GPIOs

If there's no sleep or reset GPIOs, video_bridge_set_active() returns
-ENOENT. Don't fail in this case, since these GPIOs are optional.

Signed-off-by: Vasily Khoruzhick <anarsoul@gmail.com>
---
 drivers/video/bridge/anx6345.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/bridge/anx6345.c b/drivers/video/bridge/anx6345.c
index 3e3f0e2ce6..28cf2a9c2d 100644
--- a/drivers/video/bridge/anx6345.c
+++ b/drivers/video/bridge/anx6345.c
@@ -274,7 +274,7 @@ static int anx6345_enable(struct udevice *dev)
 
 	/* Deassert reset and enable power */
 	ret = video_bridge_set_active(dev, true);
-	if (ret)
+	if (ret && ret != -ENOENT)
 		return ret;
 
 	/* Reset */
-- 
2.18.1

